xx = persona('aaa')
xx.hablar()
del xx 
xx.hablar() >>>> error ya que dice que no existe
